torch/csrc/utils/tensor_memoryformats.h

```diff
@@ -1,13 +1,13 @@
 #pragma once
 
 #include <c10/core/MemoryFormat.h>
-#include <torch/csrc/utils/pybind.h>
+#include <torch/csrc/utils/python_stub.h>
 
 namespace torch {
 namespace utils {
 
 void initializeMemoryFormats();
-py::object getTHPMemoryFormat(c10::MemoryFormat);
+PyObject* getTHPMemoryFormat(c10::MemoryFormat);
```

Comment:
Is this because you need this file in libtorch? Or just to avoid all the release().ptr() at the call sites?

Reply:
It's a header cycle if it uses pybind11